src/sfdt/populate.ts

```diff
@@ -70,7 +71,8 @@ export default (data, sfdt, prefix = 'DATA::') => {
 					dataMode = true
 				}
 
-				currentlyProcessing = inline.name
+				// process bookmark name
+				currentlyProcessing = last(inline.name.split("::"))
```

Comment:
this lib should NOT know about the way we use our bookmarks. we should not hardcode it to a naming style like '::', else it will not be re-usable for others.